test/e2e/resize_nodes.go

```diff
@@ -101,7 +104,8 @@ func groupSize() (int, error) {
 }
 
 func waitForGroupSize(size int) error {
-	for start := time.Now(); time.Since(start) < 4*time.Minute; time.Sleep(5 * time.Second) {
+	timeout := 4 * time.Minute
```

Comment:
I was going to suggest you draw these constants out to the top of the file.... but I really don't know whether it matters. Up to you.